ext/debug/ctf/bottoms/posix/ctf_bottom.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <misc/__assert.h>
+#include "ctf_bottom.h"
+
+ctf_bottom_ctx_t ctf_bottom;
+
+
+void ctf_bottom_configure()
+{
+	ctf_bottom.pathname = "zephyr_trace.ctf";
```

Comment:
Nicety: I guess a command line option defaulting to this would be nice for users.